### PR TITLE
atdgen 1.7.0 release

### PR DESCRIPTION
**Pull request by mjambon**

### CODE REVIEW

**Comment by samoht** [comment]
@mjambon can you try to fix the `opam-lint` errors? Thanks!

File: packages/atdgen/atdgen.1.7.1/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mjambon/atdgen/archive/v1.7.0.tar.gz"
+checksum: "7d2065ef55cfae5861508e6869a024d4"
```
Comment (dsheets) [inline packages/atdgen/atdgen.1.7.1/url]: Looks like the archive URL and the checksum don't match for 1.7.1.

Comment (mjambon) [inline packages/atdgen/atdgen.1.7.1/url]: Thanks, I forgot to update the URL.

**Comment by dsheets** [comment]
This looks good but the `.install` file is unnecessary, I believe. The archive already contains a `.install` file and due to <https://github.com/ocaml/opam/blob/master/src/state/opamAction.ml#L73> the basename of the files listed in it will be used during opam installation. I believe this has been true since at least opam 1.1.

**Comment by avsm** [comment]
Duplicated `install` file can be fixed later...

**Pull request merged**